lib/screens/HomeScreen.tsx

```diff
@@ -568,119 +591,207 @@ const HomeScreen: React.FC<HomeScreenProps> = ({ navigation, route }) => {
+                styles.userPhoto,
+                { backgroundColor: theme.black },
+              ]}
+              onPress={() => {
+                navigation.navigate("AccountPage");
+              }}
+            >
+              <Text style={styles.pfpText}>{userInitials}</Text>
+            </TouchableOpacity>
+            <Image source={require('../../assets/icon.png')} style={{width: width * 0.105, height: width * 0.105, marginEnd: width * 0.025}} />
+    
+            <View>
+            <Tooltip
+              isVisible={tooltipVisible}
+              content={
+                <View>
+                  <Text style = {styles.buttonText}>Search Entries Here!</Text>
+                  <Button
+                    style={styles.gotItButton}
+                    onPress={() => {
+                      setTooltipVisible(false);
+                      setAddNoteTooltipVisible(true);
+                    }}
+                  >
+                    <Text style={styles.gotItButtonText}>Got it!</Text>
+                  </Button>
+                </View>
+              }
+              placement="bottom"
+              onClose={() => {
+                setTooltipVisible(false);
+                setAddNoteTooltipVisible(true);
+              }}
+            >
+    
+                <TouchableOpacity 
+                  onPress={() => {
+                    setIsSearchVisible(!isSearchVisible);
+                    resetSearchQuery();              
+                  }}
+                >
+                  <Ionicons
+                    testID="searchButton"
+                    name={isSearchVisible ? "close-outline" : "search-outline"}
+                    size={36}
+                    color={theme.black}
+                  />
+                </TouchableOpacity>
+           </Tooltip>
+            </View>
+          </View>
+        </View>
+    
+        <View style={styles.dropdown}>
+        <Tooltip
+          isVisible={dropDownTip}
+          content={
+            <View>
+              <Text style = {styles.buttonText}>Switch between private and published entries here!</Text>
+              <Button
+                    style={styles.gotItButton}
+                    onPress={() => setDropDownTip(false)}
+                  >
+                    <Text style={styles.gotItButtonText}>Got it!</Text>
+                  </Button>
+            </View>
+          }
+          placement="bottom"
+          arrowSize={{ width: 10, height: 10 }} // Adjust arrow size for better alignment
+          onClose={() => setDropDownTip(false)}
+          showChildInTooltip={false}>
+          <DropDownPicker
+            open={open}
+            value={value}
+            items={items.filter(item => item.value !== value)}
+            setOpen={setOpen}
+            setValue={(callback: (arg0: string) => any) => {
+              const newValue = callback(value);
+              setValue(newValue);
+              handleFilters(newValue);
```

Comment:
items for creating the tutorial looks proper with the everything labeled. add more comments to give better understanding of the functioning specifically